tests/test_api.py

```diff
@@ -14,17 +28,14 @@ def orsay_museum(es_client):
     index_name = 'munin_poi_specific'
     es_client.indices.create(index=index_name)
     es_client.indices.put_alias(name='munin_poi', index=index_name)
-    filepath = os.path.join(os.path.dirname(__file__), 'fixtures', 'orsay_museum.json')
-    with open(filepath, "r") as f:
-        orsay_museum = json.load(f)
-        poi_id = orsay_museum['id']
-        es_client.index(index='munin_poi',
-                        body=orsay_museum,
-                        doc_type='poi',
-                        id=poi_id,
-                        refresh=True)
-        return poi_id
+    return load_poi('orsay_museum.json', es_client)
 
+@pytest.fixture(scope="module")
+def blancs_manteaux(es_client):
+    """
+    fill elasticsearch with one poi, the church des blancs manteaux
+    """
+    return load_poi('blancs_manteaux.json', es_client)
```

Comment:
this breaks test encapsulation as now If you run only the test `test_block_null` the test won't pass.

To fix this, you need to create:

```python 
@pytest.fixture(scope="module")
def init_indicies(es_client):
   .. stuff from above
```

and inject it in `orsay_museum` and `blanc_manteau`

Reply:
Wops you'r right, thanks!